irc/client.go

```diff
@@ -318,6 +318,10 @@ func (server *Server) RunClient(conn clientConn, proxyLine string) {
 	session.idletimer.Initialize(session)
 	session.resetFakelag()
 
+	for _, defaultMode := range config.Accounts.defaultUserModes {
```

Comment:
Until we get support for persisting user modes of always-on clients in #819, can you copy and paste this block into `AddAlwaysOnClient` (the other code path that creates a new client)?

Reply:
Done.